libs/s25main/PickedMovableObject.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2005 - 2023 Settlers Freaks (sf-team at siedler25.org)
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "DrawPoint.h"
+#include "gameTypes/MapCoordinates.h"
+
+class GameWorldView;
+
+class PickedMovableObject
+{
+public:
+    static PickedMovableObject Pick(const GameWorldView& gwv, MapPoint mapPt, DrawPoint drawPt, bool expire);
```

Comment:
In new code I tried to use `UpperCamelCase` for classes and `lowerCamelCase` for methods so that one can easily see where a class is used and where a method is called. Would you mind following that style?

(Yes due to historical growth this is largely inconsistent)

Reply:
Perfect.  I generally tried to adapt to the surrounding code, but the inconsistencies and lack of direction were bothering me. (I was going to suggest adding `CONTRIBUTING.md` and maybe `CODE_STYLE.md`, which could be part of `CONTRIBUTING.md` as well, but you get the point.)

Reply:
Fully agreed. It's just that nobody want's to write that ;-)